doc/nrf/ug_thread_configuring.rst

```diff
@@ -190,13 +190,10 @@ Select one of the following logging levels to customize the logging output:
 * :kconfig:option:`CONFIG_OPENTHREAD_LOG_LEVEL_INFO` - This option additionally enables informational logging.
 * :kconfig:option:`CONFIG_OPENTHREAD_LOG_LEVEL_DEBG` - This option additionally enables debug logging.
 
-The more detailed logging level you select, the more logging buffers you need to be able to see all messages.
-The buffer size must also be increased.
-Use the following Kconfig options for this purpose:
-
-* :kconfig:option:`CONFIG_LOG_STRDUP_BUF_COUNT` - This option specifies the number of logging buffers.
-* :kconfig:option:`CONFIG_LOG_STRDUP_MAX_STRING` - This option specifies the size of logging buffers.
+The more detailed logging level you select, the bigger logging buffer you need to have to see all messages.
+Use the following Kconfig option for this purpose:
 
+* :kconfig:option:`CONFIG_LOG_BUFFER_SIZE` - This option specifies the number of logging buffers.
```

Comment:
It seems it's not exactly the number of buffers: _Number of bytes dedicated for the logger internal buffer_